### PR TITLE
Added file to convert csv to json

### PR DESCRIPTION
**Pull request by Yolo86**
__1__ 

#1 

#### Short description of what this resolves:
Task 1 completed
the python file in Submissions/Yolo86/Task_1/ will convert csv file to json
the json file is included in the folder
...
Modified code to choose all movies with comedy genre
...
Fixed json file and modified python script(relative path)

#### Changes proposed in this pull request and/or Screenshots of changes:

-
-
-

### CODE REVIEW

File: Submissions/Yolo86/Task_1/csvtojson.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+with open('C:\\Users\\SAKSHAM\\Desktop\\OpenCode20\\2-Python t1\\Monty_Python_Class\\Task_1\\movies.csv','r', encoding="utf8") as csv_file:
+	fieldnames = ("movieId","title","genres")
+	csv_reader = csv.DictReader(csv_file,fieldnames)
+
+	with open('csv2json.json','w', encoding="utf8") as json_file:
+
+		for line in csv_reader:
+
+			if line['genres'] == 'Comedy':
```
Comment (deepakDOTexe) [inline Submissions/Yolo86/Task_1/csvtojson.py]: There can be movies whose genre contains other genres also apart from only 'Comedy' such as 'Drama' or 'Action' so make your code more flexible.

Comment (Yolo86) [reply Submissions/Yolo86/Task_1/csvtojson.py]: Alright..I'll make the required changes

**Comment by Yolo86** [comment]
fixed

File: Submissions/Yolo86/Task_1/csv2json.json [hunk 2]
```diff
@@ -0,0 +1,18780 @@
+{
+    "movieId": "1",
+    "title": "Toy Story (1995)",
+    "genres": "Adventure|Animation|Children|Comedy|Fantasy"
+}
+{
```
Comment (deepakDOTexe) [inline Submissions/Yolo86/Task_1/csv2json.json]: This is not the correct json format. Please use a dictionary to store all the rows containing comedy movies and then dump that dictionary into a json file.

**Comment by Yolo86** [comment]
@deepakDOTexe fixed now

File: Submissions/Yolo86/Task_1/csvtojson.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import csv
+import json
+import os
+
+path1 = os.getcwd()
+for i in range(3):
+	path1 = os.path.split(path1)[0]
+path1 = os.path.join(path1,"","Task_1\movies.csv")
```
Comment (Lokesh2703) [inline Submissions/Yolo86/Task_1/csvtojson.py]: use os.path.join() for joining Task_1\movies.csv

Comment (Yolo86) [reply Submissions/Yolo86/Task_1/csvtojson.py]: I used that(os.path.join()) to join it..

**Pull request closed without merging**